### ISSUE TITLE
Patch for musl's utmp.h is incomplete

### ISSUE THREAD

**Issue by dslm4515** [dslm4515/CMLFS#92]
Per [MLFS's closed issue](https://github.com/dslm4515/Musl-LFS/issues/54), musl's `utmp.h` should also be patched, not just `path.h`

Looks like the patch imported from MLFS is missing half of it [patch for utmps.h]

**Comment by takusuman** [comment]
To be honest, I can not remember if I did this on Copacabana too.
I mean, we haven't sourced the patch directly from Musl-LFS, we have made the changes per ourselves.
I'll take a look at the documentation that I've written in the last year... Good thing I'm working on it now.

**Comment by takusuman** [comment]
Eh, just looked at it, we haven't changed and committed the same error. Nice.
I'll be taking this fix for Copacabana too. Merci for the observation.

**Comment by dslm4515** [comment]
I only noticed it when running the ssh daemon:
```
wtmp_write: problem writing /dev/null/wtmp: Not a directory
```
But then after applying the full patch, I still have this error:
```
wtmp_write: problem writing /run/utmps/wtmp: No such file or directory
```
... which is no surprise:
```
$ ls -a /run/utmps/
.  ..  .utmpd-socket  .utmpd-socket.lock  .wtmpd-socket  .wtmpd-socket.lock  utmp
```
as for some reason my boot script for `wtmpd` is not creating a wtmp file

**Comment by takusuman** [comment]
> I only noticed it when running the ssh daemon:

Maybe that's another reason for why Heirloom NG tools that depend on utmp weren't working at Copacabana 0.4. With this fix, they will finally do now.

**Comment by dslm4515** [comment]
Fixed with commit 160ee2af858050f1e0204b039f3908c2ef54f467
